policy-test/src/lib.rs

```diff
@@ -435,7 +435,7 @@ where
         drop(_tracing);
     }
 
-    if std::env::var("POLICY_TEST_NO_CLEANUP").is_err() {
+    if std::env::var("POLICY_TEST_NO_CLEANUP").is_ok() {
```

Comment:
is this right? it seems backwards.  POLICY_TEST_NO_CLEANUP being ok means we should NOT delete

Reply:
leftover